commercetools-models/src/test/java/io/sphere/sdk/orders/commands/OrderImportCommandIntegrationTest.java

```diff
@@ -350,6 +349,19 @@ public void customerId() throws Exception {
     }
 
     @Test
+    public void inventoryMode() throws Exception {
+        testOrderAspect(builder -> {
+                    builder.inventoryMode(null);
+                },
+                order -> assertThat(order.getInventoryMode()).isEqualTo(InventoryMode.NONE));
+
+        testOrderAspect(builder -> {
+                    builder.inventoryMode(InventoryMode.NONE);
+                },
+                order -> assertThat(order.getInventoryMode()).isEqualTo(InventoryMode.NONE));
```

Comment:
This could be true even if we forgot to send the inventory mode with the builder. Using e.g `InventoryMode.RESERVE_ON_ORDER` will make a better assertion ;)